lib/routes/caixin/latest.ts

```diff
@@ -55,6 +56,10 @@ async function handler(ctx) {
                 } else {
                     item.description = desc.description;
                 }
+
+                if (item.appContentId) {
+                    item.description += `<div style="text-align:center;"><a href="${`https://datayi.cn/1lnZaaido8xd?id=${item.appContentId}&article_type=1&isHttp=0&type=%E6%99%AE%E9%80%9A%E6%96%87%E7%AB%A0%E9%A1%B5&open_type=6`}">打开App</a></div>`;
```

Comment:
This site involves in ads and tracking activities and has been listed in multiple AdBlock lists.

> growingio.com

https://adguardteam.github.io/HostlistsRegistry/assets/filter_1.txt
https://adguardteam.github.io/HostlistsRegistry/assets/filter_34.txt

> datayi.cn

https://easylist.to/easylist/easyprivacy.txt